dev/PowerNotifications/PowerNotifications.h

```diff
@@ -7,8 +7,71 @@
 #include <powersetting.h>
 #include <Microsoft.Windows.System.Power.PowerManager.g.h>
 #include <frameworkudk\PowerNotificationsPal.h>
-#include <wil\resource.h>
-#include <wil\result_macros.h>
+#include <..\WindowsAppSDK_Insights\WindowsAppRuntimeInsights.h>
+
+class PowerNotifications : public wil::TraceLoggingProvider
+{
+    IMPLEMENT_TRACELOGGING_CLASS(PowerNotifications, "Microsoft.WindowsAppSDK.System.PowerNotifications",
+        // {a1b12e2c-12d9-564e-2ea1-2894ffcc7cc5}
+        (0xa1b12e2c, 0x12d9, 0x564e, 0x2e, 0xa1, 0x28, 0x94, 0xff, 0xcc, 0x7c, 0xc5));
+
+    public:
+
+        DEFINE_COMPLIANT_TELEMETRY_EVENT_STRING(RegisterEvent, PDT_ProductAndServiceUsage, value);
+        DEFINE_COMPLIANT_TELEMETRY_EVENT_STRING(UnregisterEvent, PDT_ProductAndServiceUsage, value);
+
+        DEFINE_COMPLIANT_MEASURES_ACTIVITY(EnergySaverStatusCallbackActivity, PDT_ProductAndServicePerformance);
```

Comment:
> DEFINE_COMPLIANT_MEASURES_ACTIVITY [](http://example.com/codeflow?start=7&length=35)

Why are these activities and not just events?

Reply:
Because they are at the boundary of external/OS API calls and we wish to log it if it fails